Gruntfile.js

```diff
@@ -90,7 +93,9 @@ module.exports = function(grunt) {
               'node_modules/jquery-ui-dist/jquery-ui.css',
               'app/css/main.css'],
 
-          'app/css/pdf.min.css': 'app/css/pdf.css'
+          'app/css/pdf.min.css': 'app/css/pdf.css',
+
+          'app/css/bootstrap-markdown.min.css': 'node_modules/bootstrap-markdown/css/bootstrap-markdown.min.css'
```

Comment:
I prefer to take the unminified version as input.

Reply:
Yeah that's the problem: There's no unminified version included ;-) We can compile one out of the provided less file, but that'll require grunt-contrib-less, which pulls in ~180 dependencies. No problem to do that though, if it's ok for you.

Reply:
Changed